Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -647,7 +648,6 @@ extension NavigatorIndex {
                     availabilityToID[entryIDs] = newID
                     availabilityIDs[newID] = entryIDs
                     availabilityID = newID
-                    try navigatorIndex.availability.put(key: newID, value: entryIDs)
```

Comment:
Actually putting the availability information into the LMDB database was moved to the `finalize()` function where the rest of the database writing logic is.